operations/network/management_network/1.2_to_1.3_upgrade.md

```diff
@@ -0,0 +1,163 @@
+# Management Network Upgrade CSM 1.2 to 1.3
+
+- [Management Network Upgrade CSM 1.2 to 1.3](#management-network-upgrade-csm-12-to-13)
+  - [Prerequisites](#prerequisites)
+  - [Backup Switch Configurations](#backup-switch-configurations)
+  - [Mellanox](#mellanox)
+  - [Dell](#dell)
+  - [Aruba Spine](#aruba-spine)
+  - [Aruba Leaf and Leaf BMC](#aruba-leaf-and-leaf-bmc)
+
+## Prerequisites
+
+- System is currently running CANU-generated CSM 1.2 configurations.
+- Switch configurations have been generated for CSM 1.3.
+  - [Generate Switch Configurations](generate_switch_configs.md)
+  - Ensure that all site [custom configurations](https://github.com/Cray-HPE/canu/#generate-switch-configs-including-custom-configurations) have been generated.
+- CANU installed with version 1.6.7 or greater.
+  - Run `canu --version` to see version.
+  - Update the CANU RPM from the release tarball. For more information, see [Update CANU From CSM Tarball](canu/update_canu_from_csm_tarball.md)
+
+For this procedure, log in to `ncn-m001` as an administrative user.  The login should be over the site connection (`lan0`), not the `CMN`, to prevent loss of connectivity to the system.
+
+## Backup Switch Configurations
+
+- (`ncn-m001#`) Use CANU to backup the switch running configurations.  Enter the switch administrative password when prompted.
+
+```bash
+mkdir switch-upgrades-csm-1.3
+cd switch-upgrades-csm-1.3
+canu backup network --folder 1.2
+```
```

Comment:
This block should probably be indented to be under the bullet point:
```suggestion
  ```bash
  mkdir switch-upgrades-csm-1.3
  cd switch-upgrades-csm-1.3
  canu backup network --folder 1.2
  ```
```